src/main/kotlin/com/terraformation/backend/seedbank/model/ViabilityTest.kt

```diff
@@ -124,7 +124,7 @@ data class ViabilityTestModel(
         when {
           staffResponsible.isNullOrBlank() -> notes
           notes.isNullOrBlank() -> "Staff responsible: $staffResponsible"
-          staffResponsible in notes -> notes
+          "Staff responsible: $staffResponsible" in notes -> notes
```

Comment:
I was thinking more along
```
"Staff responsible: " in notes -> notes
```
That way user can edit the actual staff name and not worry about us appending the staff responsible line again.
However, my suggestion could be less foolproof - maybe not ideal.